WooCommerce/WooCommerceTests/POS/ViewModels/TotalsViewModelTests.swift

```diff
@@ -22,8 +24,41 @@ final class TotalsViewModelTests: XCTestCase {
                               currencyFormatter: .init(currencySettings: .init()),
                               paymentState: .acceptingCard,
                               isSyncingOrder: false)
+        cancellables = []
     }
-    func test_isSyncingOrder() {}
+
+    override func tearDown() {
+        cancellables.forEach {
+            $0.cancel()
+        }
+        cancellables.removeAll()
+
+        super.tearDown()
+    }
+
+    func test_isSyncingOrder_when_syncOrder_called() async throws {
```

Comment:
Oh, I missed this PR!

I updated the tests in my PR as well https://github.com/woocommerce/woocommerce-ios/pull/13534/files and removed `isSyncincOrder` altogether. So this might not be needed anymore.